src/api-service/__app__/daily_events/__init__.py

```diff
@@ -17,3 +18,8 @@ def main(mytimer: func.TimerRequest) -> None:  # noqa: F841
             logging.info("stopping proxy")
             proxy.state = VmState.stopping
             proxy.save()
+
+    scalesets = Scaleset.search()
+    for scaleset in scalesets:
+        logging.info("updating scaleset configs: %s", scaleset.scaleset_id)
+        scaleset.update_configs()
```

Comment:
As part of a general incremental effort to make things more approachable: an we add a comment here and/or to `Scaleset.update_configs()` to make it more obvious what this is doing, and why it is reasonable to do on this schedule?